src/pages/FlssPrototypes/queries/testLoanData.graphql

```diff
@@ -0,0 +1,34 @@
+#import './fragments/loanCardFields.graphql'
```

Comment:
Please move this file to `/src/graphql/query/` and rename to something like `flssQuery.graphql` and then it can just maintain the default path to the loanCardFields fragment.

Reply:
thanks @mcstover ! I was trying to keep the page queries localized and separate to flss project. thanks for letting know where the files can live.